tests/tls_handler_test.c

```diff
@@ -1465,9 +1600,35 @@ static int s_tls_client_channel_negotiation_success_ecc384_fn(struct aws_allocat
     (void)ctx;
     return s_verify_good_host(allocator, s_badssl_ecc384_host_name, 443, NULL);
 }
-
 AWS_TEST_CASE(tls_client_channel_negotiation_success_ecc384, s_tls_client_channel_negotiation_success_ecc384_fn)
 
+#    ifdef _WIN32
+
+static int s_tls_client_channel_negotiation_success_ecc384_SCHANNEL_CREDS_fn(
+    struct aws_allocator *allocator,
+    void *ctx) {
+    (void)ctx;
+
+    // Force using SCHANNEL_CREDS for testing
+    aws_windows_force_schannel_creds(true);
+    s_verify_good_host(allocator, s_badssl_ecc384_host_name, 443, NULL);
+    aws_windows_force_schannel_creds(false); // reset
+    return AWS_OP_SUCCESS;
+}
+
+AWS_TEST_CASE(
+    tls_client_channel_negotiation_success_ecc384_deprecated,
+    s_tls_client_channel_negotiation_success_ecc384_SCHANNEL_CREDS_fn)
+#    endif
+
+AWS_STATIC_STRING_FROM_LITERAL(s_aws_ecc384_host_name, "127.0.0.1");
+static int s_tls_client_channel_negotiation_success_tls1_3_fn(struct aws_allocator *allocator, void *ctx) {
+    (void)ctx;
+    return s_verify_good_host_mqtt_connect(allocator, s_aws_ecc384_host_name, 59443, NULL);
+}
+
+AWS_TEST_CASE(tls_client_channel_negotiation_success_tls1_3, s_tls_client_channel_negotiation_success_tls1_3_fn)
```

Comment:
Trivial: I think this is testing MTLS so it should probably reflect that in the test name.